R/rope.R

```diff
@@ -74,7 +94,7 @@ rope.numeric <- function(posterior, range = "default", ci = .90, verbose = TRUE)
     .rope(posterior, range = range, ci = i, verbose = verbose)
   })
 
-  out <- flatten_list(rope_values)
+  out <- do.call(rbind, rope_values)
```

Comment:
@DominiqueMakowski `flatten_list()` seems to return a data frame with _list_ columns in this case. I don't think this is the expected behaviour, so I changed this code.